src/gui/MixerChannelView.cpp

```diff
@@ -22,474 +22,457 @@
  *
  */
 
+#include "MixerChannelView.h"
+
+#include <QCheckBox>
+#include <QFont>
+#include <QGraphicsProxyWidget>
+#include <QGraphicsScene>
+#include <QMenu>
+#include <QMessageBox>
+#include <QPainter>
+#include <cassert>
```

Comment:
Wouldn't cassert come before qt headers?

Reply:
Line below.

Reply:
The formatting is based on our clang-tidy configuration, which if there are objections, we should fix ASAP.

Reply:
Ahh, i thought there's no specific formatting and all this is our convenience. It's fine then.

Reply:
And per my style guidelines, stl always comes top, and thought the project has similar styling.